packages/debug/src/browser/debug-session.tsx

```diff
@@ -73,8 +73,12 @@ export class DebugSession implements CompositeTreeElement {
         protected readonly breakpoints: BreakpointManager,
         protected readonly labelProvider: LabelProvider,
         protected readonly messages: MessageClient,
-        protected readonly fileService: FileService) {
+        protected readonly fileService: FileService,
+        protected readonly debugCP: ContributionProvider<DebugContribution>
```

Comment:
This is a breaking change which is not documented in the `changelog`.
Downstream clients who use `DebugSession` or extend it will be broken by the `constructor`.

Reply:
TBF it doesn't look like this is a public API (meant to be overriden without breakage). Still nice to add to the CHANGELOG.

Reply:
> I tested the feature using `pwa-extensionHost` and it worked correctly but I'm wondering why the standard `extensionHost` **type** is not supported yet (which is the default when generating an extension):
> 
> ```json
> {
>     "name": "Run Extension",
>     "type": "extensionHost",
>     "request": "launch",
>     "args": [
>         "--extensionDevelopmentPath=${workspaceFolder}"
>     ],
>     "outFiles": [
>         "${workspaceFolder}/out/**/*.js"
>     ],
>     "preLaunchTask": "${defaultBuildTask}"
> },
> ```

I don't know if and when extensionHost **type** worked. Currently **pwa-extensionHost** in `vscode-js-debug` builtin plugin replaced it. See:
- https://github.com/microsoft/vscode-js-debug/blob/master/OPTIONS.md
- https://github.com/microsoft/vscode-js-debug/blob/4747389c7d50639738aeb33c1e2f6e2980d2bd0f/COMMON_PROBLEMS.md#my-app-doesnt-run-using-a-node-10-release-before-around-10180

Reply:
added changelog update without breaking changes